### PR TITLE
Demo checks for v0.37

### PR DESCRIPTION
**Pull request by Shiro-Raven**

### CODE REVIEW

File: demonstrations/getting_started_with_hybrid_jobs.py [hunk 1]
```diff
@@ -373,7 +373,7 @@ def circuit(params):
 # is not as smooth as the simulator, but the minimum still is detected correctly!
 #
 
-df = pd.DataFrame(qpu_job.metrics())
+df = pd.DataFrame(job.metrics(metric_type=MetricType.ITERATION_NUMBER))
```
Comment (lillian542) [inline demonstrations/getting_started_with_hybrid_jobs.py:376]: why are we updating this to access the previous job and not the `qpu_job`?

**Review by lillian542 — COMMENTED** [review]
One question about the changes, and then of course the messaging about Lucy needs a quick update. Lmk when its ready for another look! :)

File: demonstrations/getting_started_with_hybrid_jobs.py (hunk 1 above)
Comment (Shiro-Raven) [reply demonstrations/getting_started_with_hybrid_jobs.py:376]: That's a copy-paste mistake, you are right

**Review by astralcai — APPROVED** [review]

**Review by lillian542 — APPROVED** [review]
Looks good, thanks! 🎉

**Pull request merged**